devito/finite_differences/differentiable.py

```diff
@@ -54,6 +58,14 @@ def is_Staggered(self):
     def _fd(self):
         return dict(ChainMap(*[getattr(i, '_fd', {}) for i in self._args_diff]))
 
+    @cached_property
+    def _symbolic_functions(self):
+        return [i for i in list(self._functions) if i.coefficients == 'symbolic']
```

Comment:
no need to cast ``self._functions`` to a list

Reply:
and perhaps you should return set, otherwise the output of this method will change across different python sessions (since it's relying on self._functions, which is unordered)

Reply:
Ok, will change, thanks.